lib/access-context.js

```diff
@@ -151,7 +151,7 @@ AccessContext.prototype.getAppId = function() {
  * @returns {boolean}
  */
 AccessContext.prototype.isAuthenticated = function() {
-  return !!(this.getUserId() || this.getAppId());
+  return Boolean(this.getUserId() != null || this.getAppId() != null);
```

Comment:
Why is it necessary to cast this value to `Boolean`? I believe the inner condition will be always a boolean, since we are using `!=` operators,

Reply:
Yep, that was a bit of knee-jerk replacement of the `!!`.